grpc-api/src/main/java/io/deephaven/appmode/ApplicationContext.java

```diff
@@ -0,0 +1,29 @@
+package io.deephaven.appmode;
+
+import io.deephaven.db.appmode.ApplicationState;
+
+/**
+ * This application context can be used to get access to the application state from
+ * within script applications.
```

Comment:
comment about how this is actually part of the app mode API, but lives in grpc-api, and why it doesn't behave as a dagger singleton, can't be an singleton instance